Docs/Practitioners/Profile.md

```diff
@@ -0,0 +1,215 @@
+| endDate | DateTime | The date the activity finished, if it has finished |
+| description | string | The description of the activity |
+| addressLines| string array | The address lines of where the activity has taken, or is taking, place |
+| city | string | The name of the city where the activity has taken, or is taking, place |
+| stateOrProvince | [Region](#region) | The region where the activity has taken, or is taking, place |
+| postalCode | string | The postalCode where the activity has taken, or is taking, place |
+| position | string | The person's position during the activity |
+| department | string | The department the person was located in during the activity |
+| wasEmployed | boolean | Indicates whether the person was employed during the activity |
+| hadStaffPrivilages | boolean | Indicates whether the person had staff privileges over the duration of the activity |
+| wasAffiliated | boolean | Indicates whether the person had any affiliations during the duration of the activity |
+| percentageClinical | integer | How much of the activity was clinical? |
+| percentageAdministrative | integer | How much of the activity was administrative? |
+
+## Address
+
+Holds the address information of the medical professional
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| isPrimary  | boolean | Indicates whether the provided address is the person's primary address |
+| addressType | string | Indicates what the address is for. Ex: Home, Business |
+| lines | string array | The address lines of the address |
+| city | string | The city name of the address |
+| stateOrProvince| [Region](#region)| The state or province of the address|
+| postalCode | string | The postal code of the address |
+
+## Ecfmg
+
+Holds information regarding the practitioner's Ecfmg certification
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| ecfmgId | string | The practitioner's ecfmg ID |
+| IssueDate | DateTime | The issue date of the certification |
+
+## EmailAddress
+
+Holds a person's email address
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| isPrimary  | boolean | Indicates whether the provided email address is the person's primary email address |
+| email | string | The person's email address |
+
+## FifthPathway
+
+Holds information regarding the practitioner's Fifth Pathway certification
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| school | [FifthPathwaySchool](#fifthPathwaySchool) | The school the practitioner attended to receive their FifthPathway certification |
+|startDate| DateTime | The date the practitioner started attendance at the Fifth Pathway school |
+|endDate |DateTime| The date the practitioner finished attending the Fifth Pathway school |
+|certificateDate | DateTime |The date the practitioner received their Fifth Pathway certification |
+
+## FifthPathwaySchool
+
+Holds information regarding a specified medical school
```

Comment:
Fix the wording like earlier but be specific - "specified medical school" -> "Fifth Pathway medical school".